sr_robot_launch/launch/srhand.launch

```diff
@@ -76,9 +77,17 @@
 
   <!-- Trajectory controller -->
   <group if="$(arg hand_ctrl)">
-    <node name="$(arg hand_id)_trajectory_controller" pkg="sr_utilities" type="trajectory_controller_spawner.py" output="screen">
-      <param name="hand_trajectory" value ="$(arg hand_trajectory)"/>
-      <param name="wait_for" value ="/calibrated" unless="$(arg sim)"/>
-    </node>
+    <group unless="$(arg grasp_controller)">
+      <node name="$(arg hand_id)_trajectory_controller" pkg="sr_utilities" type="trajectory_controller_spawner.py" output="screen">
+        <param name="hand_trajectory" value ="$(arg hand_trajectory)"/>
+        <param name="wait_for" value ="/calibrated" unless="$(arg sim)"/>
+      </node>
+    </group>
+    <group if="$(arg grasp_controller)">
+      <node name="$(arg hand_id)_grasp_controller" pkg="sr_utilities" type="grasp_controller_spawner.py" output="screen">
+        <param name="hand_trajectory" value ="$(arg hand_trajectory)"/>
```

Comment:
I couldn't see this parameter being read from the grasp controller spawner

Reply:
Yep, missed it. Fixed in 3951986